section2/exercises/if-statements.js

```diff
@@ -82,15 +91,20 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+var cupsOfFlour = 3;
 var hasSauce = true;
 
 // Write your conditional here
+if (cupsOfFlour >= 2 && hasSauce === true) {
```

Comment:
Really nice solutions on this topic, and nice style/syntax (use of whitespace, spaces, and indentation) - may seem small but it's HUGE for readability!